upgrade/overview.md

```diff
@@ -46,7 +46,7 @@ To upgrade the Telerik UI for Blazor components used in your project, perform th
 
 1. If you are using a local theme stylesheet in `wwwroot`, then replace it. For example, if the app is using a [custom theme]({%slug themes-custom%}), then recreate it with the [ThemeBuilder]({%slug themebuilder%}). Or, if the [project was created with the Telerik&reg; UI for Blazor Visual Studio Extensions]({%slug themes-swatches%}#visual-studio-and-visual-studio-code-wizards), then [download the required built-in theme]({%slug themes-swatches%}#swatch-urls) and add it to the project.
 
-1. If the application is localized, [update the Telerik localization (`.resx`) files]({%slug globalization-localization%}#how-localization-works-in-the-telerik-components). Otherwise you may see [exceptions related to missing localization strings]({%slug common-kb-null-value-parameter-format%}).
+1. If the application is localized, [update the Telerik localization resource (`.resx`) files]({%slug globalization-localization%}). Otherwise you may see [exceptions related to missing localization strings]({%slug common-kb-null-value-parameter-format%}).
```

Comment:
```suggestion
1. If the application is localized, [update the Telerik localization resource (`.resx`) files]({%slug globalization-localization%}#step-2-add-resouce-files). Otherwise you may see [exceptions related to missing localization strings]({%slug common-kb-value-cannot-be-null-parameter-format%}).
```